src/main/resources/META-INF/plugin.xml

```diff
@@ -227,6 +212,27 @@
             <!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
+
+        <action id="GitMachete.NextBranchCheckoutAction"
+                class="com.virtuslab.gitmachete.frontend.actions.traverse.NextBranchCheckoutAction">
+            <add-to-group group-id="GitRepositoryActions" anchor="last"/>
```

Comment:
Hmmm is `GitRepositoryActions` some group that we control? or is it some existing group from git4idea? 🤔

Reply:
The `OpenGitMacheteTab` actions is as well added to that. Don't we need it here?

Reply:
Huh @mkondratek do you remember what's the actual effect of having a `add-to-group` for `GitRepositoryActions`? I.e. where is the action displayed (that it otherwise wouldn't be)? 🤔

Reply:
oh, this one is interesting - gimme a moment 🕵️

Reply:
ok, I see know. `GitRepositoryActions` used to be the action group available from menu `Git`. 
**Now, OpenMacheteTabAction is NOT available in that menu.**
git4idea must have change it in the meantime. We even was expecting that but forgot it; we have the comment 
```
<!-- In future IntelliJ releases this group-id will probably be `Git.MainMenu`
  We can consider `VcsGroups` that seems to be more stable and place this action in VCS menu -->
```

I ve tested it, and the following:
```
 <add-to-group group-id="VcsGroups" anchor="last"/>
```
works fine now - as expected.

I am opening a separate issue for this one as the lack of our action in Git menu is a bug.

Reply:
So I guess this implies that I can remove `addToGroup` from the actions of this PR.